README.adoc

```diff
@@ -0,0 +1,27 @@
+= Container Quickstarts by Red Hat's Community of Practice
+
+This repository is meant to help bootstrap users of OpenShift Container Platform get started in building and using Source-to-Image to build applications to run in OpenShift.
```

Comment:
Should this sentence be the following?

"This repository is meant to help bootstrap users of **the** OpenShift Container Platform **to** get started"